libs/features/mas/src/checkout.js

```diff
@@ -15,16 +10,23 @@ import { toOfferSelectorIds, toQuantity } from './utilities.js';
  * generate Checkout configuration
  */
 export function Checkout({ providers, settings }) {
+    // Memoize settings for frequently used values
+    const {
+        checkoutClientId,
+        checkoutWorkflow: defaultWorkflow,
+        checkoutWorkflowStep: defaultWorkflowStep,
+        country: defaultCountry,
+        language: defaultLanguage,
+        promotionCode: defaultPromotionCode,
+        quantity: defaultQuantity,
+        env, 
```

Comment:
we didn't have env and landscape before, how are the used in checkout? just wondering what changed